configure/toolchain.c

```diff
@@ -34,6 +34,23 @@ OS_API = posix
 #  else
 OS_API = score
 #  endif
+#  if defined(RTEMS_NETWORKING)
+     // legacy stack circa RTEMS <= 5 and networking internal to RTEMS
```

Comment:
I'm a little concerned about including comments like this in this particular file — see the remarks at the top. Maybe they're okay for all the RTEMS preprocessor versions, but developers on Windows might see them in the file and try to add them to other parts.

Reply:
Thanks for the review and highlighting the issue with comments on Windows. I have removed the comments.

Reply:
Comments in `toolchain.c` within `#ifdef __rtems__` should not be a problem.  As noted at the top of that file, GCC drops C comments from the output.  Please leave/return these comments as I think they add valuable information about the history of RTEMS networking which will, I can but hope, one day give someone confidence to remove it when this compatibility is no longer needed.

@anjohnson rather than removing comments from `toolchain.c`, wrap them with `#ifdef _COMMENT_` to hide them from MSVC.